src/client/App.jsx

```diff
@@ -3,51 +3,119 @@ import './App.css';
 import MovieForm from './components/MovieForm';
 import UserForm from './components/UserForm';
 
-const apiUrl = 'http://localhost:4000';
 
-function App() {
+function App () {
   const [movies, setMovies] = useState([]);
+  const [regInfo, setInfo] = useState({});
 
   useEffect(() => {
-    fetch(`${apiUrl}/movie`)
+    const options = {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+        'Authorization': 'Bearer ' + localStorage.getItem('jwt')
+      }
+    }
+    fetch(`http://localhost:4000/movie`, options)
       .then(res => res.json())
       .then(res => setMovies(res.data));
-  }, []);
+  }, [])
+
 
   const handleRegister = async ({ username, password }) => {
-    
+    const options = {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({ username, password })
+    }
+    fetch('http://localhost:4000/user/register', options)
+      .then(res => res.json())
+      .then(res => {
+        setInfo(res.data)
+      })
+      .catch(() => { setInfo('Server Error') })
+    console.log('my fetch response', regInfo)
   };
 
+
   const handleLogin = async ({ username, password }) => {
-    
+    const options = {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({ username, password })
+    }
+    fetch('http://localhost:4000/user/login', options)
+      .then(res => res.json())
+      .then(res => {
+        localStorage.setItem('jwt', res.data)
+        console.log('loged in', localStorage.getItem('jwt'))
+        console.log(res)
+
+        const options = {
+          method: 'GET',
+          headers: {
+            'Content-Type': 'application/json',
+            'Authorization': 'Bearer ' + localStorage.getItem('jwt')
+          }
+        }
+        fetch(`http://localhost:4000/movie`, options)
+          .then(res => res.json())
+          .then(res => setMovies(res.data));
+      })
+      .catch(err => console.log(err))
   };
-  
+
+
+
   const handleCreateMovie = async ({ title, description, runtimeMins }) => {
```

Comment:
All of your handle functions are very clear and concise. Very consistent and neat code :)